inc/namespace.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * SEO module functions.
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform\SEO;
+
+use const HM\Platform\ROOT_DIR;
+use function HM\Platform\get_config;
+use HM\Platform\Module;
+
+function bootstrap( Module $module ) {
+	$settings = $module->get_settings();
+
+	if ( $settings['redirects'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_redirects' );
+	}
+
+	if ( $settings['xml-sitemaps'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_sitemaps' );
+	}
+
+	if ( $settings['metadata'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_metadata' );
+	}
+
+	if ( $settings['amp'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_amp' );
+	}
+
+	if ( $settings['facebook-instant-articles'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_instant_articles' );
+	}
+
+	// Read robots.txt file into robots.txt route handled by WP.
+	add_filter( 'robots_txt', __NAMESPACE__ . '\\robots_txt', 10 );
+}
+
+function get_bool_callback( bool $condition ) : string {
+	return $condition ? '__return_true' : '__return_false';
+}
+
+function load_redirects() {
+	require_once ROOT_DIR . '/vendor/humanmade/hm-redirects/hm-redirects.php';
+}
+
+function load_sitemaps() {
+	require_once ROOT_DIR . '/vendor/humanmade/msm-sitemap/msm-sitemap.php';
+}
+
+function load_metadata() {
+	require_once ROOT_DIR . '/vendor/humanmade/wp-seo/wp-seo.php';
+	require_once ROOT_DIR . '/vendor/humanmade/meta-tags/plugin.php';
+
+	$config = get_config()['modules']['seo']['metadata'] ?? [];
+
+	// Enable / disable plugin features.
+	add_filter( 'hm.metatags.twitter', get_bool_callback( $config['twitter'] ?? false ) );
```

Comment:
Same here for not needing null coalesce

Reply:
We're not merging the arrays at this level so it could be omitted. I could flatten the settings out to avoid that situation.